sonar-helm-for-iac/converters/tree_converter.go

```diff
@@ -0,0 +1,283 @@
+package converters
+
+import (
+	org_sonarsource_iac_helm "github.com/SonarSource/sonar-iac/sonar-helm-for-iac/org.sonarsource.iac.helm"
```

Comment:
I'll change it to `pbstructs` to better indicate its purpose. I won't change the generated package name itself; it can be done in another protobuf-related PR